src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/redundantimport/InputRedundantImportWithoutPackage.java

```diff
@@ -0,0 +1,13 @@
+/*
+RedundantImport
+
+
+*/
+
+import com.puppycrawl.tools.checkstyle.checks.imports.redundantimport.*;
```

Comment:
Please explain why we need this import.
My concern is extension of suppression of `id="noCheckstyleInInputs`